tests/integration_tests/test_aggregated_data_example.py

```diff
@@ -1,21 +1,32 @@
 import sys
 import subprocess  # nosec
 
+from pathlib import Path
+from dash.testing.composite import DashComposite
```

Comment:
```suggestion
import sys
import subprocess  # nosec
from pathlib import Path

from dash.testing.composite import DashComposite
```
Relevant in some of the other files as well.